src/title.ts

```diff
@@ -16,10 +21,17 @@ export interface TitleBase {
    */
   offset?: number;
 
+  /**
+   * A [mark style property](config.html#style) to apply to the title text mark.
+   *
+   * __Default value:__ `"group-title"`.
+   */
+  style?: string | string[];
+
   // TODO: name, encode, interactive, zindex
 }
 
-export interface Title extends TitleBase {
+export interface TitleParams extends TitleBase {
```

Comment:
Why the rename? It doesn't seem to semantically fit as well, given that we don't use "Params" in other, similar cases